CSS folder/index.css

```diff
@@ -0,0 +1,269 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Times New Roman", Times, serif;
+  line-height: 1.4;
+}
+
+a {
+  text-decoration: none;
+}
+
+p {
+  margin: 0.75rem 0;
+}
+.text-primary {
+  color: #ffa500;
+}
+
+.navbar {
+  display: flex;
+  background: #333;
+  color: #fff;
+  position: sticky;
+  top: 0;
```

Comment:
Is this required? what happens if you omit this?

Reply:
I actually want the navbar to stick to the browser's top but I just noticed it can be omitted.